clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/DcosConfigurationProperties.groovy

```diff
@@ -40,4 +51,25 @@ class DcosConfigurationProperties {
   static class LinkedDockerRegistryConfiguration {
     String accountName
   }
+
+  public static Config buildConfig(final Account account, final Cluster cluster, final ClusterCredentials clusterConfig) {
```

Comment:
There a reason these methods were moved here? Doesn't seem like this is the right place for them, since this is strictly supposed to be the class used for mapping to the spring account config.